src/reporters/HTMLReporter.ts

```diff
@@ -35,7 +36,7 @@ export class HTMLReporter implements IReporter {
     else reportPath = <string>this.argumentsProvider.html;
 
     await this.fileSystemService.writeFile(path.resolve(process.cwd(), reportPath), reportHTML);
-    console.log('Report was saved to ' + reportPath);
+    debug.log('Report was saved to ' + reportPath);
```

Comment:
`debugLog `?

Reply:
debugLog() doesn't log in terminal. debug.log() logs both in the terminal and in the file if the -l is used.